modules/nf-core/yahs/main.nf

```diff
@@ -0,0 +1,39 @@
+process YAHS {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::yahs=1.2a.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/yahs:1.2a.2--h7132678_0':
+        'quay.io/biocontainers/yahs:1.2a.2--h7132678_0' }"
+
+    input:
+    tuple val(meta), path(bed)
+    path fasta
+    path fai
+
+    output:
+    tuple val(meta), path("*scaffolds_final.fa") , emit: scaffolds_fasta
+    tuple val(meta), path("*scaffolds_final.agp"), emit: scaffolds_agp
+    tuple val(meta), path("*bin")                , emit: binary
```

Comment:
The output files aren't being detected, are you sure this can find them all?

Reply:
I think the test.yml just needs updating for the new prefix ('test')